docker/e2e/cluster.mongodb.yml

```diff
@@ -0,0 +1,53 @@
+version: '3.4'
```

Comment:
3.7? I had some issues with 3.4 when specifying complex `entrypoint` & `command` values in d-c.

Reply:
Sure.